libs/platform/src/lib/form/combobox/combobox/combobox.component.scss

```diff
@@ -20,6 +20,16 @@
             span {
                 overflow: hidden;
                 text-overflow: ellipsis;
+                &.left-aligned {
+                    // special fix for IE, in which left-aligned span may jump to the right when tabbed over
+                    margin-right: auto;
+                    
+                    [dir="rtl"] &,
+                    &[dir="rtl"] {
+                        margin-right: 0;
+                        margin-left: auto;
+                    }
+                }
             }
```

Comment:
Should we add these changes in fundamental-style instead of fundamental-ngx?

Reply:
@manu-kr the issue appears to be specific to fundamental-ngx library, hence no reason to put these styles in `fundamental-styles` repo